### ISSUE TITLE
[20211128] Weekly AI ArXiv 만담

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#31]
- News
  - [2021 과학기술미래인재포럼 영상](https://www.youtube.com/watch?app=desktop&v=31QnsY2vrAM)
  - [2021 AI대학원 심포지엄](https://www.aigs.kr/default/) --> 신청은 홈피에서
    - 12월 3일 코엑스에서 개최됩니다.
  - [2021 ICT트렌드 컨퍼런스 by IT여성기업인협회](http://www.kibwa.org/board/board_view?code=notice&no=1127)
    - 장소는 코엑스 역시 날짜는 12월 3일
    - 참가신청: https://forms.gle/zCFgShoi1uxemmWj9 
  - [DEVIEW 2021 다시보기](https://deview.kr/2021/sessions)
  
- Arxiv
  - [DeBERTaV3: Improving DeBERTa using ELECTRA-Style Pre-Training with Gradient-Disentangled Embedding Sharing](https://arxiv.org/abs/2111.09543)
    - MS 365 AI와 MSR의 DeBERTa 멤버들이 그대로 개선
    - DeBERTa: Disentangled attention (word content and position): https://arxiv.org/pdf/2006.03654.pdf (ICLR 2021), 공개는 작년 6월 --> RoBERTa보다 좋은 성능 보임
    - MLM을 Electra의 pretraining task인 repace token detection (RTD)로 대체 (대체 생성된 token 잡아내기)
    - Electra 스타일로 하되 Electra의 D와 G 사이의 tug of war (줄다리기) 문제 해결을 위한 gradient-disengtangled embedding sharing 제안
    - https://github.com/microsoft/DeBERTa
![image](https://user-images.githubusercontent.com/11782739/143729483-de1e15a7-5a06-4218-bd45-14941d0f6574.png)
![image](https://user-images.githubusercontent.com/11782739/143729518-4b8c8df1-764e-4254-b7b0-669923992649.png)
![image](https://user-images.githubusercontent.com/11782739/143729536-c535c6b4-87d8-4b60-9678-0df4aa050ce8.png)

  - [GauGAN2](https://blogs.nvidia.com/blog/2021/11/22/gaugan2-ai-art-demo/)
    - GauGAN2 combines segmentation mapping, inpainting and text-to-image generation in a single model, making it a powerful tool to create photorealistic art with a mix of words and drawings.
    
  - [NÜWA: Visual Synthesis Pre-training for Neural visUal World creAtion](https://arxiv.org/abs/2111.12417)
    - 이름이 왜 이런가 했더니: https://en.wikipedia.org/wiki/N%C3%BCwa (중국 고대신화의 여와..)
    - text, image, video를 모두 입력에 넣고 visual decoding 하는 멀티모달 foundation model (from MSRA, 북경대)
    - 이를 위해서 3D Nearby attention 기반의 변형 transformer
    - pretraining task: T2I, video prediction, T2V: 다양한 어려운 pretraining task approach
    - Image tokenizer는 VQ-GAN (Video도 그냥 그대로)
    - Pretraining data: 2.9M I-T pair, Moments in Time (727k video), VATEX (241k text-video pair)
![image](https://user-images.githubusercontent.com/11782739/143729808-738ee82b-0720-48e6-b379-788d6a35fe32.png)
![image](https://user-images.githubusercontent.com/11782739/143729816-0011bf97-a816-4713-9d54-62459d7d9a30.png)
![image](https://user-images.githubusercontent.com/11782739/143729902-f68e84da-d1ce-44c1-b7b8-16a720ea2c34.png)

  - [Florence: A New Foundation Model for Computer Vision](https://arxiv.org/abs/2111.11432)
    - 다양한 CV task들을 위한 Foundation model (Foundation model이 궁금한 분은: ) from MSR and MS Cloud
    - 훈련데이터는 web scale image-text data
    - 12 layer transformer (LM encoder) + Conv+SwinTransformer (Vision encoder)
    - Image-text contrastive learning은 UniCL을 쓴다는데 (동일 그룹) 2022년에 arxiving을 준비하고 있다는... ㅡㅡ;;;
    - Scalability를 위한 테크닉들 함께 소개 (그래서 512 A100 10일간 돌림)
    - Transfer type: finetuning, linear probing, few-shot, zero-shot for 44 benchmark tasks 
    - ImageNet-1k zero sho 83,7%, COCO 62.4 mAP, VQA 80.36, 87.8 for K-600

![image](https://user-images.githubusercontent.com/11782739/143763982-e82fe8fb-bd3f-4f86-bb1d-ab65c6cdd7b0.png)

**Comment by laray5959** [comment]
- ArXiv 
    - [One-shot Voice Conversion for Style Transfer based on Speaker Adaptation](https://arxiv.org/abs/2111.12277)
    - [Improved Prosodic Clustering for Multispeaker and Speaker-independent Phoneme-level Prosody Control](https://arxiv.org/abs/2111.10168)

**Comment by kimyoungdo0122** [comment]
- News
    - DEVIEW 2021과 HyperCLOVA(다시보기는 하정우 소장님 게시글로!)
        -  인상깊었던 부분은?
![image](https://user-images.githubusercontent.com/42539153/143734937-b6855aed-36a2-4081-a763-433ec5e3f5d2.png)
![image](https://user-images.githubusercontent.com/42539153/143734957-978cc51e-6b83-4413-84e6-162851199da8.png)
        - HyperCLOVA는 단순히 좋은 PLM이 아니라 연구와 개발, 서비스, 비즈니스 등 모든 요소를 아우르는 Lifecycle의 핵심적인 마중물
        - DEVIEW 챙겨보던 지인은 HyperCLOVA 중심의 DEVIEW에 아쉬움을 내비치기도...
        - Large-scale AI를 사내에 구축할지 말지 고민하시는 분들께 좋은 행사였을 것 같습니다
    -  [OpenAI’s API Now Available with No Waitlist](https://openai.com/blog/api-no-waitlist/)
        - [API Guidlines](https://beta.openai.com/docs/introduction)
        - [API Pricing](https://openai.com/api/pricing/#faq-playground-usage)
![image](https://user-images.githubusercontent.com/42539153/143764618-445ab597-cf11-404c-8f5f-5c6f2ef16686.png)
        - [pre-model description](https://beta.openai.com/docs/engines)
        - 바로 저번주에 Azure에 올라간다고 했는데.. 바로 api pricing을 여는 이유는?
        -

**Comment by qqueing** [comment]
- [Scaling Law for Recommendation Models: Towards General-purpose User Representations
](https://arxiv.org/pdf/2111.11294.pdf)
  - 네이버의 추천 쪽 하이퍼 스케일 유저 representations 연구
  - 다운스트림 태스크에 대한 실험 및 다양한 실험 결과들(scaling law에 대한 확인등)
  - (참고) 데뷰 2021 https://tv.naver.com/v/23649376

**Comment by hollobit** [comment]
### 유네스코 인공지능 윤리 권고 채택 
https://unesdoc.unesco.org/ark:/48223/pf0000379920.page=14
https://kaiea.org/research/?idx=8994590&bmode=view
https://zdnet.co.kr/view/?no=20211124105459
>- 유네스코 193개 회원국이 인공지능 윤리에 관한 첫 번째 글로벌 협약을 채택
>- 141개 조항의 가치와 원칙 

### 정부가 만든 'AI 윤리원칙 자율점검표' 실효성 놓고 의견 분분
http://it.chosun.com/site/data/html_dir/2021/11/26/2021112601857.html
http://www.aitimes.com/news/articleView.html?idxno=141704
- AI 윤리 자율점검표 초안에서는 작년 12월 과기정통부가 마련한 AI 윤리기준 내 10대 핵심 요건에 해당하는 체크리스트 문항 47개를 제시
- 하지만 자율점검표의 실효성을 둘러싼 업계의 반응은 나뉜다. 실무에서 활용할 수 있는 구체적인 지침이 필요하다는 의견이 있지만, 자율적 지침이 아닌 규제적 성격을 갖게 될까 우려하는 목소리도 있다. 또 자율적 지침이기에 기업들이 적극 활용하지 않을 것이란 우려도 있다
![141704_144436_3054](https://user-images.githubusercontent.com/998803/143768535-3ae757a0-5f50-4334-a274-4c36b06db223.jpg)
![141704_144437_313](https://user-images.githubusercontent.com/998803/143768546-992e555d-3f79-4957-b7a3-09f46f74cce6.jpg)
